contracts/tests/SpigotedLine.t.sol

```diff
@@ -1088,5 +1088,38 @@ contract SpigotedLineTest is Test {
       vm.prank(lender);
       line.claimAndRepay(address(revenueToken), tradeData);
     }
+
+    // tests that the amount used to repay the lender cannot cause an underflow
+    function test_lender_use_and_repay_underflow() public {
```

Comment:
I haven t tried foundry invairant testing yet but it might be better to do an invariant test on `_repay()` in Loc.t.sol which would cover all repayment functions

Reply:
I'll do it as an additional